tests/rustdoc-ui/scrape-examples-fail-if-type-error.stderr

```diff
@@ -5,7 +5,7 @@ LL |   INVALID_FUNC();
    |   ^^^^^^^^^^^^ could not resolve path `INVALID_FUNC`
    |
    = note: this error was originally ignored because you are running `rustdoc`
-   = note: try running again with `rustc` or `cargo check` and you may get a more detailed error
```

Comment:
Made more sense for this to be a help instead of a note, but I can save these stylistic changes for later.